website/blog/2022-12-19-Build-Your-First-Hudi-Lakehouse-with-AWS-Glue-and-AWS-S3.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Build Your First Hudi Lakehouse with AWS S3 and AWS Glue"
+excerpt: "Follow this tutorial on building your first hudi lakehouse with AWS S3 & AWS Glue"
+author: Soumil Shah
```

Comment:
Nadine, can we change this to your name  since you are authoring this blog?

Reply:
sure- updating